docs/portal/developer-portal/operations/CVE_2024_0461.md

```diff
@@ -0,0 +1,90 @@
+
+# Mitigation for CVE-2023-0461
+
+## Description
+
+ There is a use-after-free vulnerability in the Linux Kernel which can be exploited to achieve local privilege escalation. To reach the vulnerability, kernel configuration flag CONFIG_TLS or CONFIG_XFRM_ESPINTCP has to be configured, but the operation does not require any privilege. There is a use-after-free bug of icsk_ulp_data of a struct inet_connection_sock. When CONFIG_TLS is enabled, users can install a tls context (struct tls_context) on a connected tcp socket. The context is not cleared if this socket is disconnected and reused as a listener. If a new socket is created from the listener, the context is inherited and vulnerable. The setsockopt TCP_ULP operation does not require any privilege. We recommend upgrading past commit 2c02d41d71f90a5168391b6a5f2954112ba2307c
+
+[CVE-2023-0461](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-0461)
+
+Current status from SUSE:
+https://www.suse.com/security/cve/CVE-2023-0461.html
+
+## Mitigation
+
+While the underlying CVE is being addressed by SUSE, UANs can mitigate this issue by unbonding the CAN (if it is being used), and unloading the TLS kernel module after blocking the kernel from being loaded again.
+
+The mitigation script provided below will perform the following actions:
+
+1. Select the first `BONDING_SLAVE0` from `ifcfg-bond0` if it exists
+1. Replace `bond0` in `ifcfg-can0` with the `BONDING_SLAVE0` interface
+1. Reload interfaces
+1. Add a blacklist for the tls module
+1. Unload `bonding` and `tls` kernel modules provided `mlx5_core` is not present
+1. Fail if `mlx5_core` is detected to highlight that the mititagtion failed
+
+*Important*: This mitigation is intend for UANs that meet the following criteria:
```

Comment:
Do we want "Important" in italics or bold? I thought it was in bold in the other PRs.

Reply:
Fixed to bold.